src/modules/likeItems.js

```diff
@@ -0,0 +1,39 @@
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/ilWLjX7DiufqJ1ygUhcb/likes';
+const getnumberofLikes = async () => {
+  const cards = document.querySelectorAll('.cards');
+  const likeCount = document.querySelectorAll('.like-count');
+  await fetch(url)
+    .then((response) => response.json())
+    .then((json) => {
+      cards.forEach((card, index) => {
+        json.forEach((item) => {
+          if (item.item_id === card.id) {
+            likeCount[index].innerHTML = item.likes;
+          }
+        });
+      });
+    });
+};
+const Interactlikebutton = async () => {
+  const likeBtns = document.querySelectorAll('.like-btn');
+  const likeBtnIcon = document.querySelectorAll('.fa-heart');
+  const likeBtnCount = document.querySelectorAll('.like-count');
+  const cards = document.querySelectorAll('.cards');
+  likeBtns.forEach((btn, index) => {
+    btn.addEventListener('click', () => {
+      likeBtnIcon[index].classList.remove('fa-regular');
+      likeBtnIcon[index].classList.add('fa-solid');
+      likeBtnCount[index].innerHTML = Number(likeBtnCount[index].innerHTML) + 1;
+      fetch(url, {
+        method: 'POST',
+        body: JSON.stringify({
+          item_id: cards[index].id,
+        }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      });
```

Comment:
- [ ] Well done implementing interactions on the like icon and being able to like items but after clicking on the like icon the number of likes increases by 2 instead of 1 on the UI so make sure that you fix it
   See in the video below :point_down: 
   
  [Screencast from 03-25-2023 10:28:08 PM.webm](https://user-images.githubusercontent.com/92303891/227745376-ca1d7e76-f0e6-4d0f-907e-27b145858f58.webm)

Reply:
Many thanks! Can you provide a video that I can watch, as the one above cannot open.